wwwroot/test/csv/regionMapping.json

```diff
@@ -4,6 +4,7 @@
     "SA1": {
       "layerName": "region_map:FID_SA1_2011_AUST",
       "server": "http://regionmap-dev.nationalmap.nicta.com.au/region_map/ows",
+      "serverType": "WMS",
```

Comment:
yeah it'd be nice if this were the default so we didn't have to add this everywhere across all apps.